### DISCUSSION TITLE
No such command: KONG_DATABASE=postgres

### DISCUSSION THREAD

**Discussion by a-jahanshahlo** [Kong/kong#11932] [Help]
I'm newbie on Kong

I want to enable database mode with below command:
`KONG_DATABASE=postgres docker-compose --profile database up`

but show me error?!
I'm using windows prompt command

`No such command: KONG_DATABASE=postgres`

**Comment by StarlightIbuki** [comment]
Windows does not support inline environment variable definitions. You could refer to: https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/set_1
